src/main/java/model/player/Players.java

```diff
@@ -0,0 +1,55 @@
+package model.player;
+
+import static java.util.stream.Collectors.collectingAndThen;
+import static java.util.stream.Collectors.toList;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class Players {
+    private static final int MIN_SIZE_OF_PLAYERS = 2;
```

Comment:
```suggestion
    private static final int MIN_PLAYERS_SIZE = 2;
```
전반적으로 자연스럽게 읽히는 코드를 작성하시기 위해 많은 노력을 하신 것 같아요 👍 
불필요한 단어를 줄이면서 이를 달성해보는건 어떨까요?
예를 들어 
* `validateSizeOfPlayers` 라는 메서드는 `validatePlayersSize`
* `getSizeOfPlayers` 라는 메서드는 `getPlayersSize` 가 될 수 있겠네요!

Reply:
리팩토링 완료했습니다!

`'Of' 와 같은 문법을 지켜야 하는가? vs 직관적인 표현법이 나은가?` 에 대한 고민이 있었습니다.
문법을 지켜야 한다로 기준을 세웠었는데, 문법을 지키다보면 변수명이 불필요하게 길어져 오히려 가독성이 떨어지는 것 같습니다.
제안 감사합니다!!

Reply:
저도 문법을 지키는게 더 잘 읽히겠지? 라는 생각을 했었는데 요즘엔 의미를 잘 전달하면 괜찮다는 생각으로 바꼈어요. 길게 표현해야 그 의미를 나타낼 수 있다면 어쩔 수 없지만, 짧게 써도 나타낼 수 있다면 그렇게 쓰는게 대부분의 사람들이 읽기 좋다고 하더라구요 🙂